Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version: 5.6
```

Comment:
We could lower this but I don't know how far down makes sense in practice, given that Xcode 14 is more or less mandatory since Ventura